src/map/clif.c

```diff
@@ -9963,6 +9963,9 @@ void clif_parse_ActionRequest_sub(struct map_session_data *sd, int action_type,
 				return;
 			}
 
+			if( sd->block_action & PCBLOCK_SITSTAND ) // *pcblock script command
```

Comment:
Just a nitpick, but when you edit this to use a bitfield, could you please make your whitespace consistent with newly written code? We're trying to stick to a spacing conform to `if (condition) {` in all new code.